test/mpi/coll/redscatbkinter.c

```diff
@@ -59,7 +59,7 @@ int main(int argc, char **argv)
         }
 
         for (i = 0; i < sendcount; i++) {
-            sendbuf[i] = (long long) (rank * sendcount + i);
+            sendbuf[i] = (long long) rank *(long long) sendcount + i;
```

Comment:
I don't think the second cast is necessary now that the parens are removed.

Reply:
Nitpick aside this is probably fine to merge as it is.

Reply:
> 
> 
> I don't think the second cast is necessary now that the parens are removed.

Agreed. Will update.